Dockerfile

```diff
@@ -1,21 +1,53 @@
 ############################################################
 ## Dockerfile to build the ga4gh server on mod_wsgi-express
 ## Configurable to use a local dataset
-## Based on mod_wsgi-docker
-## Results of this build are available from Dockerhub as afirth/ga4gh_server_apache:prod
 ############################################################
-FROM grahamdumpleton/mod-wsgi-docker:python-2.7-onbuild
+FROM ubuntu
 
-# File Author / Maintainer
-MAINTAINER Alastair Firth
+# Originally created by Steve Hershman GitHub @hershman
```

Comment:
I in principle, but I ended up not reusing any of Alastair's code, as far as I could tell.